tests/phpunit/sql-outputs/scd4-snfk-keep-del-active.sql

```diff
@@ -0,0 +1,75 @@
+-- SCD4: This method snapshot full current state of the data to the snapshot table. --
+
+-- The start and end dates DO NOT contain the time ("use_datetime" = false). --
+SET CURRENT_DATE = (SELECT CONVERT_TIMEZONE('UTC', current_timestamp()))::DATE;
+
+SET CURRENT_DATE_TXT = (SELECT TO_CHAR($CURRENT_DATE, 'YYYY-MM-DD'));
+
+-- Last state: Actual snapshot of the all rows  --
+CREATE TABLE "last_state" AS
+    SELECT
+        -- Monitored parameters. --
+        input."Pk1" AS "pk1", input."pk2" AS "pk2", input."name" AS "name", input."Age" AS "age", input."Job" AS "job",
```

Comment:
... a mame tu kopec SQL suborov, pre kazdu sadu vstupnych parametrov, ... netreba to kontrolovat, testovali sme to rucne, ... je to tu pre buduce zmeny + istota, ze to nespadne.